tensorflow/python/kernel_tests/batch_matmul_op_test.py

```diff
@@ -267,9 +267,7 @@ def benchmarkBatchMatMulBroadcast(self):
 
 if __name__ == "__main__":
   dtypes_to_test = [np.float16, np.float32, np.float64, np.int32]
-  if not test.is_built_with_rocm():
-    # ROCm does not support BLAS operations for complex types
-    dtypes_to_test += [np.complex64, np.complex128]
+  dtypes_to_test += [np.complex64, np.complex128]
```

Comment:
This apply to multiple locations. I believe the upstream used one line to specify `dtypes_to_test`. Could we align with the upstream's format?